docs/sources/auth/planning/index.md

```diff
@@ -85,7 +85,38 @@ Consider the following scenarios:
 
 These are just a few examples of how Grafana can be used in M2M scenarios. The platform is highly flexible and can be used in a variety of other M2M applications, making it a powerful tool for organizations looking to gain insights into their systems and devices.
 
-### 🚧 Service Accounts
+### Service accounts
+
+You can use a service account to run automated workloads in Grafana, such as dashboard provisioning, configuration, or report generation. Create service accounts and service accounts tokens to authenticate applications, such as Terraform, with the Grafana API.
+
+> **Note:** Service accounts will eventually replace [API keys]({{< relref "../api-keys/" >}}) as the primary way to authenticate applications that interact with Grafana.
+
+A common use case for creating a service account is to perform operations on automated or triggered tasks. You can use service accounts to:
+
+- Schedule reports for specific dashboards to be delivered on a daily/weekly/monthly basis
+- Define alerts in your system to be used in Grafana
+- Set up an external SAML authentication provider
+- Interact with Grafana without signing in as a user
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}), you can also use service accounts in combination with [role-based access control]({{< relref "../roles-and-permissions/access-control/" >}}) to grant very specific permissions to applications that interact with Grafana.
+
+> **Note:** Service accounts can only act in the organization they are created for. If you have the same task that is needed for multiple organizations, we recommend creating service accounts in each organization.
+
+{{< vimeo 742056367 >}}
```

Comment:
nit: is this a typo? Or is this an actual  video about service accounts?

Reply:
That's the actual video, however I dunno how to render it locally.

Reply:
You can render the docs locally in order to see what the video looks like.

```
cd docs
make docs
```
![image](https://user-images.githubusercontent.com/2051016/222711652-f8945cc4-0d7a-411b-b0fd-da5b061775a5.png)